static/src/javascripts/lib/cookies.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+const getShortDomain = (): string => {
+    // Trim subdomains for prod (www.theguardian), code (m.code.dev-theguardian) and dev (dev.theguardian, m.thegulocal)
+    const domain = document.domain || '';
+    return domain.replace(/^(www|m\.code|dev|m)\./, '.');
+};
+
+const getDomainAttribute = (): string => {
+    const shortDomain = getShortDomain();
+    return shortDomain === 'localhost' ? '' : ` domain=${shortDomain};`;
+};
+
+const remove = (name: string, currentDomainOnly: boolean = false): void => {
+    // Remove cookie, implicitly using the document's domain.
+    document.cookie = `${name}=;path=/;expires=Thu, 01 Jan 1970 00:00:01 GMT;`;
+    if (!currentDomainOnly) {
+        // also remove from the short domain
+        document.cookie = `${name}=;path=/;expires=Thu, 01 Jan 1970 00:00:01 GMT; domain=${getShortDomain()};`;
```

Comment:
Can we extract the `expires=Thu, 01 Jan 1970 00:00:01 GMT;` into a variable, so the minfier can do better work?